be/src/exec/hash_join_components.cpp

```diff
@@ -70,15 +71,15 @@ void HashJoinBuilder::reset(const HashTableParam& param) {
 
 void HashJoinBuilder::reset_probe(RuntimeState* state) {
     _key_columns.clear();
-    _ht.reset_probe_state(state);
+    (void)_ht.reset_probe_state(state);
```

Comment:
how about change the return type of JoinHashTable::reset_probe_state to void directly ? it always return OK now.